tests/plugins/colordialog/colordialog.js

```diff
@@ -0,0 +1,121 @@
+/* bender-tags: editor */
```

Comment:
So we discussed this once again, if unit tag is not needed (not used by bender) it should be skipped.